bin/sender.py

```diff
@@ -37,14 +38,20 @@ def main():
     op.add_option('-c', '--config', help='location of config file',
                   default='/etc/apel/sender.cfg')
     op.add_option('-l', '--log_config',
-                        help='location of logging config file (optional)',
-                        default='/etc/apel/logging.cfg')
-    (options, unused_args) = op.parse_args()
+                  help='DEPRECATED - location of logging config file (optional)',
+                  default=None)
+
+    options, unused_args = op.parse_args()
+
+    # Deprecating functionality.
+    old_log_config_default_path = '/etc/apel/logging.cfg'
+    if (os.path.exists(old_log_config_default_path) or options.log_config is not None):
+        logging.warning('Separate logging config file option has been deprecated.')
 
     cp = ConfigParser.ConfigParser({'use_ssl': 'true'})
     cp.read(options.config)
 
-    ssm.agents.logging_helper(cp, options.log_config)
```

Comment:
As with the receiver, does this line need to be restored?

Reply:
Path check added to sender.py and receiver.py so we don't need to pass through the path in log_config.